Dockerfile

```diff
@@ -1,8 +1,11 @@
-#FROM python:3.6-alpine
-FROM python:3.9.6-buster
+# FROM python:3.9.6-buster
```

Comment:
There's a little more we can do for "better" Dockerfiles.  That should probably be separate from this PR.  I have filed an issue #108 to investigate that.